src/components/tasks/TaskList/TaskList.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+import Card from '../card'
+import task from '@/interfaces/task.type'
+import beautifyTaskStatus from '@/helperFunctions/beautifyTaskStatus'
+import {
+  SEE_MORE,
+  INITIAL_TASKS_LIMIT,
+  ADD_MORE_TASKS_LIMIT,
+} from '../constants'
+import styles from '../card/card.module.scss'
+
+type TaksListProps = {
+  tasks: task[];
+  isEditable?: boolean;
+  hasLimit?: boolean;
+  updateCardContent?: (id: string, cardDetails: task) => void;
+};
+
+function getFilteredTasks({hasLimit, tasksLimit, tasks}: {hasLimit: boolean, tasksLimit: number, tasks: task[]}) {
```

Comment:
can we create a type for the parameters

Reply:
Done, added type `FilterTasksProps`. Check this [commit](https://github.com/Real-Dev-Squad/website-status/commit/3c011b13e671793b2880dca0a71fd45ea74ec6f2) for refference.